frontstage/templates/secure-messages/conversation-list.html

```diff
@@ -22,6 +22,14 @@
       {% endif %}
     {% endwith %}
 
+    <nav class="sm-nav u-mb-l">
+      <h2 class="u-vh">Messages navigation</h2>
+        <ul class="nav-pills">
+          <li class="nav-pills__pill{% if not is_closed %}--active {% endif %}"><a class="pill-label" href="{{ url_for('secure_message_bp.view_conversation_list') }}">Open</a></li>
+          <li class="nav-pills__pill{% if is_closed %}--active {% endif %}"><a class="pill-label" href="{{ url_for('secure_message_bp.view_conversation_list', is_closed='true') }}">Closed</a></li>
+        </ul>
+    </nav>
+
     {% if not messages %}
       <p>No new messages</p>
```

Comment:
This doesn't make sense under the closed messages tab, might be better as "No closed conversations"?